buildkite/resource_team_member.go

```diff
@@ -133,7 +179,7 @@ func (tm *teamMemberResource) Read(ctx context.Context, req resource.ReadRequest
 	}
 
 	// Convert fron Node to getNodeTeamMember type
-	if teamMemberNode, ok := apiResponse.GetNode().(*getNodeNodeTeamMember); ok {
+	if teamMemberNode, ok := r.GetNode().(*getNodeNodeTeamMember); ok {
```

Comment:
Ohh this is missing the state removal if not found. Can you add that while youre here @james2791

Reply:
Oh yep on it 👍